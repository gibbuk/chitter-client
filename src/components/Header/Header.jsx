import { NavLink } from 'react-router-dom';

const Header = () => {
    return (
        <>
            <div className="container">
                <header className="d-flex flex-wrap justify-content-center py-3 mb-4 border-bottom">
                    <NavLink to="/" className="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-dark text-decoration-none">
                        <span className="fs-4">Chitter</span>
                    </NavLink>

                </header>
            </div>
        </>
    )
}

export default Header;