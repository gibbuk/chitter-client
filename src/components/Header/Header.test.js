import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Header from './Header';

describe(`Header tests`, () => {
    test(`should match the snapshot`, () => {
        const rendered = render(
            <MemoryRouter>
                <Header />
            </MemoryRouter>
        )

        expect(rendered).toMatchSnapshot();
    });
});