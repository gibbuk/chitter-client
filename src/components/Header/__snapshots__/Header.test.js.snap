// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests should match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <header
          class="d-flex flex-wrap justify-content-center py-3 mb-4 border-bottom"
        >
          <a
            aria-current="page"
            class="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-dark text-decoration-none active"
            href="/"
          >
            <span
              class="fs-4"
            >
              Chitter
            </span>
          </a>
        </header>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <header
        class="d-flex flex-wrap justify-content-center py-3 mb-4 border-bottom"
      >
        <a
          aria-current="page"
          class="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-dark text-decoration-none active"
          href="/"
        >
          <span
            class="fs-4"
          >
            Chitter
          </span>
        </a>
      </header>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
